types/pg/index.d.ts

```diff
@@ -73,31 +70,33 @@ export declare class Pool extends events.EventEmitter {
     connect(): Promise<Client>;
     connect(callback: (err: Error, client: Client, done: () => void) => void): void;
 
-    end(callback?: () => void): Promise<void>;
+    end(): Promise<void>;
+    end(callback: () => void): void;
 
     query(queryStream: QueryConfig & stream.Readable): stream.Readable;
-    query(queryTextOrConfig: string | QueryConfig): Promise<QueryResult>;
-    query(queryText: string, values: any[]): Promise<QueryResult>;
-
+    query(queryConfig: QueryConfig): Promise<QueryResult>;
+    query(queryText: string, values?: any[]): Promise<QueryResult>;
```

Comment:
Added. But there may still be some other missing properties or methods, since the purpose of this PR is to fix definition error rather then make it complete.